mobile/status.go

```diff
@@ -2172,7 +2172,7 @@ func numberToHex(numString string) string {
 }
 
 func Sha3(str string) string {
-	return "0x" + abi_spec.Sha3(str)
+	return abi_spec.Sha3WithHexPrefix(str)
```

Comment:
I actually find `"0x" + ` format much simpler to read 🤔 
cc @status-im/status-go-guild any opinion or guidelines?